launch_ros/launch_ros/actions/node.py

```diff
@@ -171,9 +171,15 @@ def __init__(
         A yaml file with the resulting parameters from the dictionary will be
         written to a temporary file, the path to which will be passed to the
         node.
-        Multiple dictionaries/files can be passed: each file path will be
-        passed in in order to the node (where the last definition of a
-        parameter takes effect).
+
+        Multiple parameter dictionaries/files can be passed: each file path
+        will be passed in in order to the node (where the last definition of
+        a parameter takes effect).
+        However, fully qualified node names override wildcards even when
+        specified earlier.
+        If `namespace` is not specified, dictionaries are prefixed by a
+        wildcard namespace (`/**`) and other specific parameter declarations
+        may overwrite it.
```

Comment:
@felixdivo consider merging this last paragraph with the paragraph above about `dict` instances being written to a temporary file.